rift/encoding/ttypes.py

```diff
@@ -1,4 +1,11 @@
-
```

Comment:
One more comment: I just noticed that you updated the Python files that are generated from the .thrift files to the newest schema, but you did not (yet) update the .thrift files themselves which are also checked in to the repo.